NeuroID/TrackerClassExtensions/UIClassExtensions/UIViewController.swift

```diff
@@ -143,32 +145,34 @@ internal extension UIViewController {
     @objc func neuroIDViewDidAppear() {
         neuroIDViewDidAppear()
 
+        if ignoreLists.contains(nidClassName) { return }
```

Comment:
this is the duplicate check?

Reply:
Yes and no, we do a similar check on LN 157, but that is checking the children of the current view, this checks the current view itself.